{"uuid": "3dc17c0b-5818-4ee8-9db8-84c9b45c3504", "children": ["faa5637b-ae45-421c-b1c1-20c157313ed7", "5ff74448-c464-4bb7-bc0f-38e214bbcdaa"], "befores": [{"name": "supervisor_chain", "status": "broken", "statusDetails": {"message": "KeyError: 'llama3.2'\n", "trace": "  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.5/x64/lib/python3.11/site-packages/_pytest/fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/runner/work/OA/OA/tests/supervisor_chain.py\", line 19, in supervisor_chain\n    llm = get_available_providers()[model]\n          ~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^\n"}, "start": 1730427806837, "stop": 1730427807435}], "start": 1730427806837, "stop": 1730427808148}